core/tests/wdio.conf.js

```diff
@@ -302,10 +302,20 @@ exports.config = {
         console.log(`ffmpeg exited with code ${code}`);
       });
     }
-    // Set a wide enough window size for the navbar in the library pages to
-    // display fully.
-    browser.setWindowSize(1285, 1000);
-
+    // If the tests are running in debug mode,
+    // the mobile viewport information is passed as
+    // the 9th argument, else it is passed as the 8th argument.
+    // eslint-disable-next-line eqeqeq
+    mobileViewportArg = (args[0] == 'DEBUG=true') ? args[8] : args[7];
+
+    // eslint-disable-next-line eqeqeq
+    if (mobileViewportArg == '--params.mobile=True') {
+      browser.setWindowSize(600, 1000);
+    } else {
+      // Set a wide enough window size for the navbar in the library pages to
+      // display fully.
+      browser.setWindowSize(1285, 1000);
```

Comment:
Is the `--server-args="-screen 0, 1285x1000x24"` argument in the e2e workflow files redundant now? Ditto for protractor

Reply:
The `--server-args="-screen 0, 1285x1000x24"` is used to set the initial screen size, later the e2e test resizes this screen according to the size set in the config file. Therefore I'm not sure if this is redundant.

Reply:
Can you investigate and find out? For example, you could try removing the `--server-args="..."` from the workflow file and see if the video recordings still work

Reply:
Removing the server_args causes the video recordings in actions to not work.

Reply:
OK, sounds good